### PR TITLE
Fix aHelp relay to detect AFK Admins

### LINKED ISSUE #24331: AFK admins should be kicked or deadmin'd
## Description
Some badmins go afk while on a server which causes the relay to show the aHelps as seen/admin online, which might indicate to other admins in the relay they do not need attention. We should kick them or deadmin them so the relay shows a more accurate status of the aHelp. There is already some afk detection but it doesn't have any actions for it.

**Reproduction**
NA

**Screenshots**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by Titian3**
## About the PR
Gave Admins a separate 5min AFK timer and if someone send an aHelp and they are AFK it will send in the relay as a SOS instead of looking like the admin has seen it. Thank you @nikthechampiongr for this better idea than to kick everyone 😄
fixes #24331

## Why / Balance
I have made this be cause I went AFK for several rounds and a bunch of aHelps were missed but to the relay they looked ok.

## Technical details
Add admin detection condition to the afk detection and update the bwoink system discord relay message to take in to account if the admin is AFK or not.

## Media
Admin online/Not AFK
image

Admin online/AFK detected
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
NA

**Changelog**

:cl: Repo

ADMIN:
- tweak: AFK admins will trigger the SOS in aHelp relay.

### CODE REVIEW

**Comment by Titian3** [comment]
If someone could tell me how to make this only show on the admin changelog you will get 100 points.

**Comment by Pangogie** [comment]
Should do 10min. We do a lot of AFKing sometimes until we hear the bwoink.

**Comment by Pangogie** [comment]
I hit the wrong button. :)

**Comment by nikthechampiongr** [comment]
> If someone could tell me how to make this only show on the admin changelog you will get 100 points.

I would like my 100 points please.

🆑 Repo

ADMIN:
- tweak: AFK admins will trigger the SOS in aHelp relay.

**Comment by Titian3** [comment]
> I hit the wrong button. :)

10min now.

**Comment by nikthechampiongr** [comment]
@Titian3 Looking at the usages of GetTargetAdmins, it is also used to notify online admins for new ahelps. Did you test what afk admins see or hear when an ahelp is made while they are afk? A lot of admins have the game minimized and wait for the bwoink sound.

**Review by nikthechampiongr — CHANGES_REQUESTED** [review]
The changes to the GetTargetAdmins function make all ahelps completely invisible to afk admins. Maybe you should have a separate function for this?

**Comment by Titian3** [comment]
> The changes to the GetTargetAdmins function make all ahelps completely invisible to afk admins. Maybe you should have a separate function for this?

Yes you are right, when I first tested it, I think I saw the red aHelp from the first message, making the new function and testing now. 
🙏Thank you! 200 points have been deposited in your account!

**Comment by Titian3** [comment]
> The changes to the GetTargetAdmins function make all ahelps completely invisible to afk admins. Maybe you should have a separate function for this?

Re-tested, all looks working right now.

image

image

image

**Review by nikthechampiongr — CHANGES_REQUESTED** [review]
The GetTargetAdmins function is still used for determining the color of the message in discord. This means that it will appear green but the messages of the player will have SOS next to them. Are you sure this is intended?

**Review by nikthechampiongr — APPROVED** [review]
Looks good to me. Although the list with the active admins is never used and all the points where the function is called it just grabs the number of things in the list. I don't think that's an issue tho. Good luck with the actual maintainers.

File: Content.Shared/CCVar/CCVars.cs [hunk 1]
```diff
@@ -833,6 +833,12 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<int> NewPlayerThreshold =
             CVarDef.Create("admin.new_player_threshold", 0, CVar.ARCHIVE | CVar.REPLICATED | CVar.SERVER);
 
+        /// <summary>
+        /// How long a admin client can go without any input before being considered AFK.
```
Comment (metalgearsloth) [inline Content.Shared/CCVar/CCVars.cs:837]:
```suggestion
        /// How long an admin client can go without any input before being considered AFK.
```

**Pull request merged**